Error: ENOENT, chmod with npm install websockify

Tried npm install websockify

Using Node.js v0.10.17 download, OSX 10.8.3, get the following error: (I checked, websockify.js file doesn't get installed, I've tried running with sudo as well, dir permissions are drwxr-xr-x, other npm installs work just fine. Tried -g as well, same result.

npm ERR! Error: ENOENT, chmod '/export/node_modules/websockify/websockify.js'
npm ERR! If you need help, you may report this log at:
npm ERR! http://github.com/isaacs/npm/issues
npm ERR! or email it to:
npm ERR! npm-@googlegroups.com

npm ERR! System Darwin 12.3.0
npm ERR! command "node" "/usr/local/bin/npm" "install" "websockify"
npm ERR! cwd /export
npm ERR! node -v v0.10.17
npm ERR! npm -v 1.3.8
npm ERR! path /export/node_modules/websockify/websockify.js
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR! /export/npm-debug.log
npm ERR! not ok code 0

--- a/other/js/package.json
+++ b/other/js/package.json
@@ -8,7 +8,7 @@
     "type": "git",
     "url": "git://github.com/kanaka/websockify.git"
   },
-  "files": ["../../docs/LICENSE.LGPL-3"],
+  "files": ["../../docs/LICENSE.LGPL-3","websockify.js"],
   "bin": {
     "websockify": "./websockify.js"
   },

